commcare_connect/form_receiver/processor.py

```diff
@@ -220,6 +220,9 @@ def process_deliver_unit(user, xform: XForm, app: CommCareApp, opportunity: Oppo
             if dist.m <= 10:
                 flags.append(["location", "Visit location is too close to another visit"])
                 break
+    if access.suspended:
```

Comment:
Do we even want these to show up? I wonder if we want to ignore them instead.

Reply:
Yeah, should a suspended user be even able to submit the forms?

Reply:
This change is completely based on the Spec, I can make the change to ignore the submissions.

Reply:
Makes sense, I guess, then it's a spec question for Rama.